session_2/student_registry/db/migrate/20210114041229_create_students.rb

```diff
@@ -0,0 +1,13 @@
+class CreateStudents < ActiveRecord::Migration[6.1]
+  def change
+    create_table :students do |t|
+      t.string :name
+      t.string :roll_number
+      t.string :branch
+      t.decimal :cgpa, :precision => 10, :scale => 2
```

Comment:
While the test  passes, precision of 10 will allow saving values like: `100.00` which is obviously not correct as CGPA can be only between 0 and 10.